e2e/e2e.js

```diff
@@ -42,7 +42,9 @@ let isDockerRunning = false;
 
 // Validate CLI args.
 validateArgs(target, { possibleValues: ["es5", "module", "both"] });
-validateArgs(browser, { possibleValues: ["firefox", "chrome", "edge"] });
+validateArgs(browser, {
+  possibleValues: ["firefox", "chrome", "edge", "electron"],
```

Comment:
I'm not sure if there was a reason to leave `electron` out but I find it useful for being able to easily run tests in WSL 2 without having to install chrome. BTW the e2e stuff worked fine in WSL 2 with Docker for Desktop (Windows).